package.json

```diff
@@ -49,6 +49,7 @@
   },
   "devDependencies": {
     "ajv": "^6.10.2",
+    "api-testing": "^1.3.0",
```

Comment:
The new changes were released as api-testing 1.4.0.

Reply:
Thanks @clarakosi, `api-testing` has been updated from 1.3.0 to 1.4.0.